services/pseudo_controller/src/main/resources/application.properties

```diff
@@ -13,14 +13,24 @@
 #     WHETHER IN AN ACTION OF CONTRACT, TORT OR OTHERWISE, ARISING FROM, OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN THE SOFTWARE.
 
 ### Whether to test NCM, or to test Alcor HTTP APIs ###
-test_against_ncm = false
-
+test_against_ncm = true
+### If use_arion_agent = false, use dp-controller to program ebpf maps on wings;
+### otherwise, use arion agent to program ebpf maps on wings.
+use_arion_agent = false
 ### Test Controller NCM test Params ###
 number_of_vpcs = 10
 ports_node_one = 100
 ports_node_two = 200
 node_one_ip = 127.0.0.1
+node_one_mac = bb:ee:dd:00:00:01
 node_two_ip = 127.0.0.2
+node_two_mac = bb:ee:dd:00:00:02
```

Comment:
What nodes do "node_one_ip" and "node_two_ip" refer to? maybe we can have a better name to differentiate with compute nodes.

Reply:
@zzxgzgz

Reply:
These are fields used when Test Controller was only able to set up 2 compute nodes, they shouldn't be used now and we can remove them.

Reply:
Removed legacy variables in [f7fa684](https://github.com/futurewei-cloud/alcor/pull/749/commits/f7fa684d19769e17279be7a85d644be34dbdc6ad)